src/libstore/store-api.hh

```diff
@@ -695,6 +695,8 @@ public:
 
     const Stats & getStats();
 
+    StorePathSet exportReferences(const StorePathSet & storePaths, const StorePathSet & inputPaths);
```

Comment:
To be honest, I'm a bit way of putting more odd and ends (i.e. non-virtual methods) in the Store class (I prefer to keep "interfaces" slim like type classes.

But then again, this method doesn't have any other types it might be associated with (Store paths are "upstream" of stores), so maybe it's fine.